### PR TITLE
10.0 purchase order line sequence

### PR DESCRIPTION
**Pull request by jpoa**
It was missing a requirement, otherwise the sequence is not maintained when creating the stock.moves which causes one of the tests to fail.

### CODE REVIEW

File: purchase_delivery/model/delivery.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+    def grid_src_dest_get(self, cr, uid, ids, src_id, dest_id, context=None):
+        dest = self.pool.get('res.partner').browse(cr, uid, dest_id,
+                                                   context=context)
+        src = self.pool.get('res.partner').browse(cr, uid, src_id,
+                                                  context=context)
+        for carrier in self.browse(cr, uid, ids, context=context):
+            for grid in carrier.grids_id:
+                get_id = lambda x: x.id
+                country_ids = map(get_id, grid.country_ids)
```
Comment (gurneyalex) [inline purchase_delivery/model/delivery.py:49]: country_ids = grid.country_ids.ids

File: purchase_delivery/model/delivery.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            cr, uid, ids, ['name'], context)]
+
+    def grid_src_dest_get(self, cr, uid, ids, src_id, dest_id, context=None):
+        dest = self.pool.get('res.partner').browse(cr, uid, dest_id,
+                                                   context=context)
+        src = self.pool.get('res.partner').browse(cr, uid, src_id,
+                                                  context=context)
+        for carrier in self.browse(cr, uid, ids, context=context):
+            for grid in carrier.grids_id:
+                get_id = lambda x: x.id
```
Comment (gurneyalex) [inline purchase_delivery/model/delivery.py:48]: to remove, not useful

File: purchase_delivery/model/delivery.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                country_ids = map(get_id, grid.country_ids)
+                state_ids = map(get_id, grid.state_ids)
```
Comment (gurneyalex) [inline purchase_delivery/model/delivery.py:50]: state_ids = grid.state_ids.ids

File: purchase_delivery/model/delivery.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                src_country_ids = map(get_id, grid.src_country_ids)
```
Comment (gurneyalex) [inline purchase_delivery/model/delivery.py:51]: src_country_ids = grid.src_country_ids.ids

File: purchase_delivery/model/delivery.py [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+                src_state_ids = map(get_id, grid.src_state_ids)
```
Comment (gurneyalex) [inline purchase_delivery/model/delivery.py:52]: src_state_ids = grid.src_state_ids.ids

File: purchase_delivery/__init__.py [hunk 6]
```diff
@@ -1,10 +1,8 @@
 # -*- coding: utf-8 -*-
+##############################################################################
 #
-#
-#    Author: Alexandre Fayolle
-#    Copyright 2013 Camptocamp SA
-#    Author: Damien Crier
-#    Copyright 2015 Camptocamp SA
+#    Copyright (C) 2015 Eficent (<http://www.eficent.com/>)
+#               <contact@eficent.com>
```
Comment (gurneyalex) [inline purchase_delivery/__init__.py:5]: what are you doing there?

**Review by gurneyalex — CHANGES_REQUESTED** [review]
please clean up the PR, there are lots of unrelated stuff in there. Why the changes in purchase_delivery?

**Comment by jpoa** [comment]
Jup, sorry, trying to merge changes from two forks and it didn't worked as
I expected, obviously.

I will clean this up.

**Pull request closed without merging**